File: app/src/main/java/com/apps/wow/droider/Feed/View/FeedView.java
package com.apps.wow.droider.Feed.View;

import com.apps.wow.droider.Model.FeedModel;
import com.arellomobile.mvp.MvpView;

public interface FeedView extends MvpView {
    void onLoadingFeed();

    void onLoadCompleted(FeedModel model, boolean clear);

    void onLoadCompleted(FeedModel model);

    void onLoadFailed();
}

File: app/src/main/java/com/apps/wow/droider/Feed/View/FeedView.kt
package com.apps.wow.droider.Feed.View

import com.apps.wow.droider.Model.FeedModel
import com.arellomobile.mvp.MvpView

interface FeedView : MvpView {
    fun onLoadingFeed()

    fun onLoadCompleted(model: FeedModel, clear: Boolean)

    fun onPopularLoadCompleted(model: FeedModel)

    fun onLoadFailed()
}